src/main/java/soot/toolkits/graph/ExceptionalUnitGraph.java

```diff
@@ -255,47 +255,50 @@ protected void initialize(ThrowAnalysis throwAnalysis, boolean omitExceptingUnit
    *         extending <code>ExceptionalUnitGraph</code>. If a <code>Unit</code> throws one or more exceptions which are
    *         caught within the method, it will be mapped to a <code>Collection</code> of <code>ExceptionDest</code>s
    *         describing the sets of exceptions that the <code>Unit</code> might throw to each {@link Trap}. But if all of a
-   *         <code>Unit</code>'s exceptions escape the method, it will be mapped to <code>null</code, rather than to a
+   *         <code>Unit</code>'s exceptions escape the method, it will be mapped to <code>null</code>, rather than to a
    *         <code>Collection</code> containing a single <code>ExceptionDest</code> with a <code>null</code> trap. (The
    *         special case for <code>Unit</code>s with no caught exceptions allows <code>buildExceptionDests()</code> to
    *         ignore completely <code>Unit</code>s which are outside the scope of all <code>Trap</code>s.)
    *         </p>
    */
   protected Map<Unit, Collection<ExceptionDest>> buildExceptionDests(ThrowAnalysis throwAnalysis) {
```

Comment:
Restructured this method a bit to avoid unnecessary work when body.getTraps() is empty and to avoid obtaining ThrowableSet.Manager.v().EMPTY repeatedly.